README.md

```diff
@@ -3,37 +3,188 @@
+
+  ```Python
+  from econml import ContinuousTreatmentOrthoForest
+  # Use defaults
+  est = ContinuousTreatmentOrthoForest()
+  # Or specify hyperparameters
+  est = ContinuousTreatmentOrthoForest(n_trees=500, min_leaf_size=10, max_depth=10, 
+                                       subsample_ratio=0.7, lambda_reg=0.01,
+                                       model_T=LassoCV(cv=3), model_Y=LassoCV(cv=3)
+                                       )
+  est.fit(Y, T, X, W)
+  treatment_effects = est.const_marginal_effect(X_test)
+    ```
+
+* [Deep Instrumental Variables](#references)
+  
+  ```Python
+  from econml import DeepIVEstimator
+
+  est = DeepIVEstimator(n_components=10, # Number of gaussians in the mixture density networks)
+                        m=lambda z, x: treatment_model(keras.layers.concatenate([z, x])), # Treatment model
+                        h=lambda t, x: response_model(keras.layers.concatenate([t, x])), # Response model
+                        n_samples=1 # Number of samples used to estimate the response
+                        )
+  est.fit(Y, T, X, Z) # Z -> instrumental variables
+  treatment_effects = est.effect(T0, T1, X_test)
+  ```
+
+
+* Bootstrap Confidence Intervals
+  ```Python
+  from econml.bootstrap import BootstrapEstimator
+
+  # Bootstrap estimator wrapper
+  boot_est = BootstrapEstimator(est, n_bootstrap_samples=10)
+  boot_est.fit(Y, T, X, W)
+  treatment_effect_interval = boot_est.const_marginal_effect_interval(X_test, lower=1, upper=99)
+  ```
+
+To see more complex examples, go to the [notebooks](https://github.com/Microsoft/EconML/tree/master/notebooks) section of the repository. For a more detailed description of the treatment effect estimation algorithms, see the EconML [documentation](https://econml.azurewebsites.net/).
+
+# For Developers
+
+You can get starting by cloning this repository. We use 
 [setuptools](https://setuptools.readthedocs.io/en/latest/index.html) for building and distributing our package.
 We rely on some recent features of setuptools, so make sure to upgrade to a recent version with
 `pip install setuptools --upgrade`.  Then from your local copy of the repository you can run `python setup.py develop` to get started.
 
-# Running the tests
+## Running the tests
 
 This project uses [pytest](https:docs.pytest.org/) for testing.  To run tests locally after installing the package, 
 you can use `python setup.py pytest`.
 
-# Generating the documentation
+## Generating the documentation
 
 This project's documentation is generated via [Sphinx](https://www.sphinx-doc.org/en/master/index.html).  To generate a local copy
 of the documentation from a clone of this repository, just run `python setup.py build_sphinx`, which will build the documentation and place it
 under the `build/sphinx/html` path.
 
 The reStructuredText files that make up the documentation are stored in the [docs directory](docs/); module documentation is automatically generated by the Sphinx build process.
```

Comment:
The docs link is broken.  Maybe `/docs` or `../docs`?

Reply:
Or the full path like the notebooks link

Reply:
Done!